### DISCUSSION TITLE
Should DictionaryTextProvider be equatable?

### DISCUSSION THREAD

**Discussion by pejato** [airbnb/lottie-ios#1792] [Ideas]
It would be very convenient if DictionaryTextProvider conformed to Equatable. Because its `values` property is internal, it is impossible to make it equatable in projects that use Lottie.

It's not a big deal to create your own DictionaryTextProvider, but then you have to deal with name collision when importing Lottie. I think it makes sense to add an equatable conformance here, as I don't believe there will be many changes to this class, and so the maintenance burden is quite low.

**Comment by calda** [comment]
Seems reasonable -- are you interested in submitting a pull request with this change?

**Reply by pejato** [reply]
Can do 👍
